.github/workflows/unit_tests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies (Common)
       run: |
-        pip install poetry tox-gh-actions coverage
+        pip install poetry tox-gh-actions coverage virtualenv==16.7.9
```

Comment:
@alalazo I'd make that `'virtualenv<20`, and add a comment above to explain why it's restricted?

Reply:
Added the comment. For the version I'd rather pin it to something we know it works. The idea is to revert the change asap. Sadly the new release of `virtualenv` 20.0.2 shows the same issue.